development.cfg

```diff
@@ -3,5 +3,11 @@ extends =
     test-plone-4.3.x.cfg
     https://raw.github.com/4teamwork/ftw-buildouts/master/plone-development.cfg
 
+parts += upgrade
+
+[upgrade]
+recipe = zc.recipe.egg:scripts
+eggs = ftw.upgrade
```

Comment:
This should already be inherited from [`plone-development.cfg`](https://github.com/4teamwork/ftw-buildouts/blob/master/plone-development.cfg#L78-L80).
What's the reason you added that?